README.md

```diff
@@ -1,2 +1,39 @@
+![Linux](https://img.shields.io/badge/Linux-FCC624?style=for-the-badge&logo=linux&logoColor=black) ![MacOS](https://img.shields.io/badge/mac%20os-000000?style=for-the-badge&logo=apple&logoColor=white)
+
 # bazel-rules-backward-cpp
 Bazel build rules for https://github.com/bombela/backward-cpp
+
+## For now we only support Linux and macOS and the support for Windows might be added in the future.
+
+Follows a "repos/deps" pattern (in order to help with recursive dependencies). To use:
+
+1. Copy `bazel/repos.bzl` into your repository at `3rdparty/bazel-rules-backward-cpp/repos.bzl` and add an empty `BUILD` (or `BUILD.bazel`) to `3rdparty/bazel-rules-backward-cpp` as well.
+
+2. Copy all of the directories from `3rdparty` that you ***don't*** already have in ***your*** repository's `3rdparty` directory.
```

Comment:
I'm confused: this repo doesn't have a `3rdparty` directory, right? So does the user have to copy anything here? (Maybe we should just drop this step)